sorc/orog_mask_tools.fd/lake.fd/enclosure_cnvx.F90

```diff
@@ -32,15 +32,17 @@ PROGRAM testenc
     END PROGRAM
 #endif
 
-
+!> Test whether a given point, p, is inside a convex spherical polygon defined with a series of n vertices
```

Comment:
Add @author and @brief tags at the file leve.

I see there is some test code here. In such cases create an issue titled something like: "recover test code in orog_mask_tools.fd/lake.fd/enclosure_cnvx.F90 as unit test"